### PR TITLE
feat: add vscode settings

### LINKED ISSUE #1072: Include VS Code settings
For a faster and more productive experience using VS Code, there's a variety of settings that can be time consuming to set for a project like FxA as well as knowing what Extensions would be most useful for each part.

VS Code supports [multi-root workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces) and [recommended extensions](https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions), so that we can include a small settings file in each dir that will help boot-strap the VS Code experience for FxA.

These settings files should also set editor defaults appropriate for the sub-package that meet FxA style guidelines and tooling (appropriate linting, etc).

### PR DESCRIPTION
**Pull request by bbangert**
Fixes issue with fxa-email-service by not building it as it has
outdated Rust crates in use.

Update docs to reflect usage of docker-ce vs. outdated docker.io.

Closes #1072.

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get -y install docker-ce \
+    #   #
+    # Java install for FxA
+    && apt-get -y install openjdk-11-jre \
+    #    #
+    # FxA dependencies.
+    && apt-get -y install build-essential git-core libgmp3-dev graphicsmagick python-virtualenv python-dev pkg-config libssl-dev \
```
Comment (shane-tomlinson) [inline .devcontainer/Dockerfile]: `git-core` is a virtual package:

> shane@shane-VirtualBox:~/development/fxa$ apt show git-core
Package: git-core
State: not a real package (virtual)
N: Can't select candidate version from package git-core as it has no candidate
N: Can't select versions from package 'git-core' as it is purely virtual
N: No packages found

Does it install correctly? When I tried that command line in a new ubuntu VM it failed and I had to install `git` instead.

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+# Configure apt and install packages
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils sudo 2>&1 \
+    #
+    # Verify git, process tools, lsb-release (common in install instructions for CLIs) installed
+    && apt-get -y install git procps lsb-release software-properties-common \
+    #
+    # Install Docker CE CLI
+    && apt-get install -y apt-transport-https ca-certificates curl gnupg-agent \
```
[the remark this answers is not in the document]
Comment (bbangert) [reply .devcontainer/Dockerfile]: https://stackoverflow.com/questions/45023363/what-is-docker-io-in-relation-to-docker-ce-and-docker-ee

tl;dr, docker.io is the older version, docker-ce is the latest and greatest free version.

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (bbangert) [reply .devcontainer/Dockerfile]: It did seem to install correctly, I'll change it to git and retry to verify that's fine as well.

**Comment by dannycoates** [comment]
I plan on trying this tomorrow 🤞
